lib/experimental/Navigation/ApplicationFrame/index.tsx

```diff
@@ -19,57 +19,74 @@ export function ApplicationFrame({ children, sidebar }: ApplicationFrameProps) {
   )
 }
 
+const SkipToContentButton = ({ contentId }: { contentId?: string }) => {
+  return (
+    <a
+      href={`#${contentId}`}
+      className={focusRing(
+        "absolute z-50 -translate-y-full translate-x-4 bg-f1-background-secondary-hover p-2 text-lg no-underline transition-transform duration-200 focus-visible:translate-y-0"
+      )}
+    >
+      Skip to content
```

Comment:
What about translations? Should we start thinking about it?

Reply:
yes, that will be the next step and a separate PR

Reply:
Okay!